etna/search/views.py

```diff
@@ -188,6 +212,11 @@ class SearchLandingView(BucketsMixin, TemplateView):
     template_name = "search/search.html"
     bucket_list = CATALOGUE_BUCKETS
 
+    def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
+        className = self.__class__.__name__
+        data["customDimension3"] = className
+        return data
```

Comment:
We can use a similar implementation in [etna.records.model](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/records/models.py#L389)s  for `customDimension3`